jinja2/base.html

```diff
@@ -119,6 +119,14 @@
       </div>
   </main>
 
+  {% block contrib_popup %}
```

Comment:
We need to invert this logic:

* No contribution popup by default
* Include popup on homepage and wiki docs page

This will keep it off the document editing page, search results, etc.